backend/README.md

```diff
@@ -0,0 +1,192 @@
+## COVID-19 Self-reporting API server User IDs
+
+
+## Requirements 
+
+You should have docker and docker-compose installed on your machine 
+
+## Installation 
+
+* clone the project from the repo 
+* cd <your_project_folder>/backend/
+* Run this command
+    ```
+    $ docker-compose up --build
+    ```
+ 
+ The node js project will run on http://localhost:4080/
+ 
+ * admin-mongo interface will be displayed on http://localhost:8082/  
+ (this is deactivated by default. You need to uncomment lines in the docker-compose.yml to activate it)
+ 
+ * Internal Mongo DB connection string
+ `mongodb://mongo/safetrace`
+
+ * External Mongo DB connection string (Robo3t)
+ `mongodb://localhost:10975/safetrace`
```

Comment:
in the `docker-compose.yml` you map port `10975` to `27017`, so shouldn't this be reflected here?

Reply:
Yes, we can explain that docker ports redirection, just I wanted to let it simple avoiding large explanation of docker stuff. I will add a brief description.